gr-zeromq/lib/base_impl.cc

```diff
@@ -72,6 +76,10 @@ base_sink_impl::base_sink_impl(int type,
 #endif
     }
 
+    // Set ZMQ_LINGER so socket won't infinitely block during teardown
+    // Re: Github #1132
```

Comment:
I'd prefer the GitHub issue reference to be in the commit message rather than the code. Otherwise this seems like a great improvement, thanks!